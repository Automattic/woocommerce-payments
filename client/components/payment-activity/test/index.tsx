/**
 * External dependencies
 */
import React from 'react';
import { render } from '@testing-library/react';

/**
 * Internal dependencies
 */
import PaymentActivity from '..';

describe( 'PaymentActivity component', () => {
	it( 'should render', () => {
		const { container } = render( <PaymentActivity /> );

		expect( container ).toMatchSnapshot();
	} );
} );
