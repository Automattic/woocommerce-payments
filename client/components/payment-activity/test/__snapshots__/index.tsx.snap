// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaymentActivity component should render 1`] = `
<div>
  <div
    class="components-surface components-card wcpay-payments-activity__card css-nsno0f-View-Surface-getBorders-primary-Card-rounded em57xhy0"
    data-wp-c16t="true"
    data-wp-component="Card"
  >
    <div
      class="css-mgwsf4-View-Content em57xhy0"
    >
      <div
        class="components-flex components-card__header components-card-header wcpay-payments-activity__card__header css-sz55ca-View-Flex-sx-Base-sx-Items-ItemsRow-Header-borderRadius-borderColor-medium-borderless em57xhy0"
        data-wp-c16t="true"
        data-wp-component="CardHeader"
      >
        Your payment activity
      </div>
      <div
        class="components-card__body components-card-body wcpay-payments-activity__card__body css-1nwhnu3-View-Body-borderRadius-medium em57xhy0"
        data-wp-c16t="true"
        data-wp-component="CardBody"
      >
        <div
          class="payments-activity-data"
        >
          <div
            class="total-payments-volume"
          >
            <div
              class="total-payments-volume__heading"
            >
              Total payments volume
              <svg
                class="gridicon gridicons-help-outline"
                height="24"
                viewBox="0 0 24 24"
                width="24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g>
                  <path
                    d="M12 4c4.411 0 8 3.589 8 8s-3.589 8-8 8-8-3.589-8-8 3.589-8 8-8m0-2C6.477 2 2 6.477 2 12s4.477 10 10 10 10-4.477 10-10S17.523 2 12 2zm4 8a4 4 0 00-8 0h2c0-1.103.897-2 2-2s2 .897 2 2-.897 2-2 2a1 1 0 00-1 1v2h2v-1.141A3.991 3.991 0 0016 10zm-3 6h-2v2h2v-2z"
                  />
                </g>
              </svg>
            </div>
            <div
              class="total-payments-volume__body"
            >
              $30.00
            </div>
          </div>
          <div
            class="other-payments-data"
          >
            COMING SOON
          </div>
        </div>
      </div>
    </div>
    <div
      aria-hidden="true"
      class="components-elevation css-91yjwm-View-Elevation-sx-Base-elevationClassName em57xhy0"
      data-wp-c16t="true"
      data-wp-component="Elevation"
    />
    <div
      aria-hidden="true"
      class="components-elevation css-91yjwm-View-Elevation-sx-Base-elevationClassName em57xhy0"
      data-wp-c16t="true"
      data-wp-component="Elevation"
    />
  </div>
</div>
`;
